commercetools-models/src/test/java/io/sphere/sdk/products/commands/updateactions/ChangeAssetNameTest.java

```diff
@@ -0,0 +1,22 @@
+package io.sphere.sdk.products.commands.updateactions;
+
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.json.SphereJsonUtils;
+import io.sphere.sdk.products.Product;
+import org.junit.Test;
+
+import static io.sphere.sdk.models.LocalizedString.ofEnglish;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ChangeAssetNameTest {
```

Comment:
We normally focus on integration tests (in this case they're located in `ProductUpdateCommandIntegrationTest`) and don't write a lot of unit tests for our update actions. And I really like this test 👍 🥇

Reply:
@katmatt oh, thanks for informing me about `ProductUpdateCommandIntegrationTest`. Didn't know about it. I will add some ITs there then ;)

Reply:
@katmatt [Integration tests](https://github.com/commercetools/commercetools-jvm-sdk/pull/1725/files#diff-bf86d04f48ca49a1342702ccf6779fcb) added for:
1. [changeAssetNameByVariantIdAndAssetKey](https://github.com/commercetools/commercetools-jvm-sdk/pull/1725/files#diff-bf86d04f48ca49a1342702ccf6779fcbR2380)
2. [changeAssetNameByVariantIdAndAssetKeyWithStaged](https://github.com/commercetools/commercetools-jvm-sdk/pull/1725/files#diff-bf86d04f48ca49a1342702ccf6779fcbR2400)
3. [setAssetCustomTypeByVariantIdAndAssetKey](https://github.com/commercetools/commercetools-jvm-sdk/pull/1725/files#diff-bf86d04f48ca49a1342702ccf6779fcbR2819)
4. [setAssetCustomTypeByVariantIdAndAssetKeyWithStaged](https://github.com/commercetools/commercetools-jvm-sdk/pull/1725/files#diff-bf86d04f48ca49a1342702ccf6779fcbR2845)
3. [setAssetCustomFieldByVariantIdAndAssetKey](https://github.com/commercetools/commercetools-jvm-sdk/pull/1725/files#diff-bf86d04f48ca49a1342702ccf6779fcbR2877)
4. [setAssetCustomFieldByVariantIdAndAssetKeyWithStaged](https://github.com/commercetools/commercetools-jvm-sdk/pull/1725/files#diff-bf86d04f48ca49a1342702ccf6779fcbR2916)

Reply:
Thanks @katmatt :) would be glad if it's merged asap.

Reply:
Ok and then we will ship it with our next release planned for this thursday.